tests/Microsoft.Identity.Test.Unit/ExceptionTests/ExperimentalFeatureTests.cs

```diff
@@ -17,18 +17,21 @@ namespace Microsoft.Identity.Test.Unit.ExceptionTests
     [TestClass]
     public class ExperimentalFeatureTests
     {
+        private static readonly string[] s_scopes = ["scope"];
```

Comment:
nit: I am generally a fan of refactoring, but it's making life harder for reviewers. Next time, pls try to separate out refactoring either in a separate PR or in a separate commit.